README.md

```diff
@@ -343,6 +343,90 @@ $ plextraktsync --server=Example1 info
 INFO     Enabled 2 libraries in Plex Server: ['Movies', 'TV Shows']
 ```
 
+### Per server configuration
+
+If you want to specify your config per server you can do so inside of `servers.yml`. Within the `config` part of the server configuration you can specify how that specific server should work.
+
+```yml
+  Example1:
+    token: ~
+    urls:
+      - http://localhost:32400
+    config:
+      sync:
+        plex_to_trakt:
+          collection: true
+        trakt_to_plex:
+          liked_lists: false
+```
+Only `sync` can be used inside of the config key-value. Keep in mind that the sync-config set in `servers.yml` overrides the global sync-config in `config.yml`.
```

Comment:
"only sync" is not true either, because other section describes "library" and "excluded-libraries".

perhaps reword as: "sync" section overrides can be used or something like that.

Reply:
That is very true. Nice catch. What do you think about: "Using `sync` in a server config overrides the global sync-config in `config.yml`." as an alternative?

Reply:
Commited an alternative! Let me know.